code/front-end/Satellite GUI.md

```diff
@@ -0,0 +1,35 @@
+# Satellite GUI
+
+Satellite GUI is a [vue.js](https://vuejs.org/) + [TypeScript](https://www.typescriptlang.org/) front-end codebase that provides an UI interface for users to interact with the network.
+
+## Development Workflow
+
+There are two ways to start local development environment depending on what you need.
+
+### Prerequisites
+
+- Clone the [storj repo](https://github.com/storj/storj)
+- Make sure you have both [node.js](https://nodejs.org/en/) and [npm](https://www.npmjs.com/get-npm) installed
```

Comment:
We should have somewhere which version is node is supported.
I would add it in the `package.json` file (https://docs.npmjs.com/files/package.json#engines) and I will inform here where to check which version should be installed.